src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapConnection.cs

```diff
@@ -634,17 +627,17 @@ private int SendRequestHelper(DirectoryRequest request, ref int messageID)
                 if (request is DeleteRequest)
                 {
                     // It is an delete operation.
-                    error = Wldap32.ldap_delete_ext(_ldapHandle, ((DeleteRequest)request).DistinguishedName, serverControlArray, clientControlArray, ref messageID);
+                    error = NativePal.LdapDeleteExt(_ldapHandle, ((DeleteRequest)request).DistinguishedName, serverControlArray, clientControlArray, ref messageID);
                 }
                 else if (request is ModifyDNRequest)
                 {
                     // It is a modify dn operation
-                    error = Wldap32.ldap_rename(_ldapHandle,
-                                                 ((ModifyDNRequest)request).DistinguishedName,
-                                                 ((ModifyDNRequest)request).NewName,
-                                                 ((ModifyDNRequest)request).NewParentDistinguishedName,
-                                                 ((ModifyDNRequest)request).DeleteOldRdn ? 1 : 0,
-                                                 serverControlArray, clientControlArray, ref messageID);
+                    error = NativePal.LdapRename( _ldapHandle,
+                                            ((ModifyDNRequest)request).DistinguishedName,
```

Comment:
I'm not sure what indentation style this is going for, but our general style is "indent, don't align".


```suggestion
                    error = NativePal.LdapRename( _ldapHandle,
                        ((ModifyDNRequest)request).DistinguishedName,
```

Or, even better (though we're less prescriptive here)

```suggestion
                    error = NativePal.LdapRename(
                        _ldapHandle,
                        ((ModifyDNRequest)request).DistinguishedName,
```

Reply:
Do you know if we have an analyzer we've used for that? The whole project has this aligning problem, so it would be nice if I could just use code refactoring to fix the indentation.